academic_research.md

```diff
@@ -0,0 +1,33 @@
+The CA can be seen as a point of failure because in the worst scenario, it can be compromised and a malicious actor could create a fake certificate and trick the user into believing that it is a real one.
+
+------------
+
+Decentralized PKI can be divided into two main categories:
+- **Log-based DPKI**: This solution is mainly used to monitor the CA. Indeed certificates are not valid until they are logged on append-only public logs. Anyone can read the logs and so monitor the CA for suspect certificates. The first log-based solution is [Certificate Transparency](https://datatracker.ietf.org/doc/html/rfc6962.html), proposed by Google.
+- **Blockhain-based DPKI**:This solution can be split into two parts:
+    - Hierarchical: it is still a partially dependent version on the CA. It can be considered as a log-based DPKI, the only difference is the use of the blockchain. For this reason this solution is called "relaxed centralization"
+    - Web of Trust: typically replace CA with miners in public blockchains, certificates are generated by mining.
+
+In the text below we will use the term **consensus** that should be defined as follows: There are various types of consensus some more efficient than others, such as:
+- *Proof of Work* (PoW): it is used in the famous Bitcoin blockchain and many other forks of Bitcoin itself. The system requires miners to compete with each other to be the first to solve "mathematical problems". The more a miner has computing power, the more likely it is to be chosen.
+- *Proof of State* (PoS): it will be used in the near future by Ethereum. PoS eliminates the problem of having a large calculation capacity in favor of a random choice of the miner.
+- [*Byzantin Fault Tolerance*](https://ieeexplore.ieee.org/document/8433150) (BFT): there are various version of it, some more efficient than others, like [HotStuff](https://dl.acm.org/doi/pdf/10.1145/3293611.3331591). The procedure can be seen in three phases: 
+	- Pre-Prepare phase: he leader initiates the consensus procedure by sending a signed pre-prepare message that is a block proposal containing a certain number of transactions.
+	- Prepare phase: Upon receiving the pre-prepare message, each node in the consensus group checks the correctness and validity of the block and multicasts a signed prepare message (yes/no) to all other nodes.
+	- Commit phase: Each node, after analysis of received prepare messages, multicasts a signed commit message (yes/no) to the consensus group. The block proposal is committed to the blockchain only if a sufficient number of nodes in the consensus group agree on it.
+
+
+
+------------
+
+There are a lot of papers about DPKI, some log-based, some blockchain-based. For this project we will focus exclusively on the Web of Trust.
```

Comment:
"a lot" is something we use in common language but not so much in writing official documents, you should chose something like: many, numerous, various.